src/status_im2/contexts/chat/messages/drawers/view.cljs

```diff
@@ -196,3 +200,66 @@
               :on-press            (fn []
                                      (rf/dispatch [:hide-bottom-sheet])
                                      (when on-press (on-press)))}]))]])))
+
+(defn get-reaction-authors-accessibility-id
```

Comment:
This function has no usages in the code.

Reply:
Removed